gallery/.eslintrc

```diff
@@ -4,6 +4,9 @@
     "require": true,
     "document": true
   },
+  "env": {
+    "jest": true
```

Comment:
I'd rather this only be active for linting the tests, which would require a separate `test-lint` script in `package.json` and a separate eslint config file.  That's complex enough that I do think this should be split to a separate ticket after all.

Reply:
> I'd rather this only be active for linting the tests, which would require a separate `test-lint` script in `package.json` and a separate eslint config file. That's complex enough that I do think this should be split to a separate ticket after all.

I removed the eslint changes for this PR and created a tech debt ticket for it.
https://issues.sonatype.org/browse/RSC-978